shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -318,25 +318,23 @@ bool FlutterWindowsEngine::Run(std::string_view entrypoint) {
     auto host = static_cast<FlutterWindowsEngine*>(user_data);
     host->view()->OnPreEngineRestart();
   };
-  args.update_semantics_node_callback = [](const FlutterSemanticsNode* node,
-                                           void* user_data) {
+  args.update_semantics_callback = [](const FlutterSemanticsUpdate* update,
+                                      void* user_data) {
     auto host = static_cast<FlutterWindowsEngine*>(user_data);
-    if (!node || node->id == kFlutterSemanticsNodeIdBatchEnd) {
-      host->accessibility_bridge_->CommitUpdates();
-      return;
+
+    for (size_t i = 0; i < update->nodes_count; i++) {
+      const FlutterSemanticsNode* node = update->nodes + i;
```

Comment:
Here and below, consider `update->nodes[i]` for readability.

Reply:
Fixed